From: SeongJae Park <sj@kernel.org>
Date: Fri, 24 Oct 2025 07:40:58 -0700
Subject: [PATCH] Docs: submitting-patches: suggest adding previous version
 links

For pre-merge and post-merge review of patches that revisioned multiple
times, patch changelogs are very useful.  Adding actual links to the
previous versions can further help double checking if the changelog is
true, and finding more discussions made before.  Implicitly suggest
adding such links if available.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/process/submitting-patches.rst | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Documentation/process/submitting-patches.rst b/Documentation/process/submitting-patches.rst
index 9a509f1a68730..e5dd5ca9683c7 100644
--- a/Documentation/process/submitting-patches.rst
+++ b/Documentation/process/submitting-patches.rst
@@ -805,7 +805,8 @@ not part of the changelog which gets committed to the git tree. It is
 additional information for the reviewers. If it's placed above the
 commit tags, it needs manual interaction to remove it. If it is below
 the separator line, it gets automatically stripped off when applying the
-patch::
+patch.  Adding links to previous version of the previous versions can
+also help reviewers double checking the changes::
 
   <commit message>
   ...
@@ -814,6 +815,9 @@ patch::
   V2 -> V3: Removed redundant helper function
   V1 -> V2: Cleaned up coding style and addressed review comments
 
+  v1: https://lore.kernel.org/foo
+  v2: https://lore.kernel.org/bar
+
   path/to/file | 5+++--
   ...
 
-- 
2.47.3

